.github/workflows/codeql-analysis.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'csharp', 'javascript' ]
+        language: [ 'javascript' ]
```

Comment:
Do we need to use the matrix here?

Reply:
It's not required, we can specify language like this https://github.com/DevExpress/devextreme-ui-template-gallery/pull/211/files#diff-63bd641104d10e25f141d518a16b22a151d125e12701df2f9e79734b23b90188R43